test_stack.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import pytest
+from stack import Stack
+
+
+def test_initialization():
+    stack = Stack()
+    assert isinstance(stack, Stack)
+    assert stack.list.head is None
+
+
+def test_push():
+    stack = Stack()
+    stack.push(5)
+    assert stack.list.head.value == 5
```

Comment:
I like that you are introspecting the list contained in the stack as a way of verifying your results.  That's good.